src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -441,7 +474,16 @@ export class AppRoot extends mixinBehaviors
             <!-- DigitalOcean servers -->
             <div class="side-bar-section servers-section" hidden\$="{{!isDigitalOceanAccountConnected}}">
               <img class="provider-icon" src="images/do_white_logo.svg">
-              <template is="dom-repeat" items="{{serverList}}" as="server" filter="_isServerManaged" sort="_sortServersByName">
+              <template is="dom-repeat" items="{{serverList}}" as="server" filter="{{_isServerForCloudProvider('DIGITALOCEAN')}}" sort="_sortServersByName">
+                <div class\$="server {{_computeServerClasses(selectedServerId, server)}}" data-server\$="[[server]]" on-tap="_showServer">
+                  <img class="server-icon" src\$="images/{{_computeServerImage(selectedServerId, server)}}">
+                </div>
+              </template>
+            </div>
+            <!-- GCP servers -->
+            <div class="side-bar-section servers-section" hidden\$="{{!isGcpAccountConnected}}">
```

Comment:
Doesn't this work?
```suggestion
            <div class="side-bar-section servers-section" hidden\$="[[!gcpAccountName]]">
```

Reply:
Also: hidden should always be a one-way bind ([[]])
https://polymer-library.polymer-project.org/2.0/docs/devguide/data-binding

Reply:
FYI, I'm fixing those issues at https://github.com/Jigsaw-Code/outline-server/pull/853.
Let's get that approved and merged so you can pull the changes.

Reply:
My PR has been submitted. You just need to pull master